appserver/tests/tck/platform-tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java

```diff
@@ -183,7 +183,7 @@ private void deleteDirectory(final Path directory) throws IOException {
     }
 
     private Process startBash(final String command) throws IOException {
-        ProcessBuilder bash = new ProcessBuilder("/bin/bash", "-c", command).inheritIO().directory(cfg.getTargetDir());
+        ProcessBuilder bash = new ProcessBuilder("/bin/zsh", "-c", command).inheritIO().directory(cfg.getTargetDir());
```

Comment:
I don't have zsh and i don;t see it as some standard ;-)

Reply:
sorry, accidentally in the commit